### ISSUE TITLE
Invalid syntax (setting meshname)

### ISSUE THREAD

**Issue by MakisH** [precice/config-visualizer#1]
I just now installed this using pip3, on my user directory. I do have Python 2 installed as well, but normally I don't have such issues.

```bash
$ precice-config-visualizer precice-config.xml 
  File "/home/makish/.local/bin/precice-config-visualizer", line 95
    meshname = f"{name}-{mesh}"
                              ^
SyntaxError: invalid syntax
```

https://github.com/precice/config-visualizer/blob/33eb9bbb1b7f95662d62762b9ed48f10ee296b09/bin/precice-config-visualizer#L95

**Comment by fsimonis** [comment]
The f format specifier is a python 3.6 feature.
Which version of python 3 are you using?

**Comment by MakisH** [comment]
Python 3.5.2. Then this needs to go into the requirements, or maybe solve it differently, if it affects more people.

**Comment by fsimonis** [comment]
True. I'll add it to the requirements and also add a runtime check.

**Comment by fsimonis** [comment]
Solved via dd0e77c
